src/lib/utils/debounce.ts

```diff
@@ -1,5 +1,4 @@
-// eslint-disable-next-line
```

Comment:
Removed some spots where we'd disabled eslint and applied TS fixes for more type safety